.github/actions/get-ocp-range/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: 'Get OCP range'
+description: 'Get the range of OCP versions corresponding to the provided range of Kubernetes versions'
+inputs:
+  kube-version-range:
+    description: 'Range of Kubernetes versions'
+    required: true
+outputs:
+  ocp-version-range:
+    description: "Corresponsing range of OCP versions"
+    value: ${{ steps.run-get-ocp-range.outputs.ocp-version-range }}
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+          go-version: '>=1.20'
+
+    - name: Install get-ocp-range
+      shell: bash
+      run: go install github.com/opdev/getocprange@latest
+
+    - name: Run get-ocp-range
+      id: run-get-ocp-range
+      shell: bash
+      run: echo "ocp-version-range=$(get-ocp-range '${{ inputs.kube-version-range }}')" >> $GITHUB_OUTPUT
```

Comment:
If `get-ocp-range` throw an error, does this fail? 

If not, lets add steps to make sure we got valid output from `get-ocp-range`. 
Bonus points if you [set an error message](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message) if it fails, or a debug (or other verbosity) message if it succeeds, with the value `get-ocp-range` returned.

Reply:
Going for the bonus point here